{"results": [{"verdicts": {"score": 100, "malicious": true, "urlscan": {"malicious": true}, "categories": ["phishing"], "hasVerdicts": true}, "submitter": {"country": "us"}, "dom": {"size": 3969, "hash": "8aedd6be2334b2cf4d998796cdd8b71154fe7cf1c129b8833ac94af913c422ff"}, "frames": {"length": 1}, "usertags": [], "canonical": {"task": {"url": "cashmaxworks868.pages.dev"}, "page": {"url": "cashmaxworks868.pages.dev"}}, "labels": ["hosting.cdn"], "task": {"visibility": "unlisted", "method": "api", "domain": "cashmaxworks868.pages.dev", "apexDomain": "cashmaxworks868.pages.dev", "time": "2024-08-19T04:31:57.953Z", "uuid": "2b3ff04d-eaa7-4976-9ff2-c96b3828ae87", "url": "https://cashmaxworks868.pages.dev/"}, "stats": {"uniqIPs": 1, "uniqCountries": 1, "dataLength": 65935, "encodedDataLength": 22141, "requests": 7}, "meta": [], "scanner": {"country": "nl"}, "links": {"length": 3}, "page": {"country": "NL", "server": "cloudflare", "ip": "188.114.96.3", "mimeType": "text/html", "title": "Worker threw exception | cashmaxworks868.pages.dev | Cloudflare", "url": "https://cashmaxworks868.pages.dev/", "tlsValidDays": 89, "tlsAgeDays": 1, "tlsValidFrom": "2024-08-17T12:02:27.000Z", "domain": "cashmaxworks868.pages.dev", "apexDomain": "cashmaxworks868.pages.dev", "asnname": "CLOUDFLARENET, US", "asn": "AS13335", "tlsIssuer": "WE1", "status": "500"}, "text": {"size": 1211, "hash": "c409c1c70f9aff5e4150bccd0fbcd738d2b6405123e75d9dc2f25141d224afa2"}, "brand": [{"country": ["us"], "name": "Generic Cloudflare", "vertical": ["Online"], "key": "genericcloudflare"}], "_id": "2b3ff04d-eaa7-4976-9ff2-c96b3828ae87", "_score": null, "sort": [1724041917953, "2b3ff04d-eaa7-4976-9ff2-c96b3828ae87"], "result": "https://urlscan.io/api/v1/result/2b3ff04d-eaa7-4976-9ff2-c96b3828ae87/", "screenshot": "https://urlscan.io/screenshots/2b3ff04d-eaa7-4976-9ff2-c96b3828ae87.png"}], "total": 2, "took": 26, "has_more": false}
